media/js/app/projects/sequenceAssignmentView.js

```diff
@@ -123,18 +123,22 @@
         isDirty: function() {
             return this.dirty;
         },
-        setDirty: function(isDirty) {
+        setDirty: function(isDirty, isSubmittable) {
```

Comment:
On second thought, I'm going to try breaking this out into two methods: `setDirty` and `setSubmittable`.